uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/microsoftsts/AadLoginComponentHandler.java

```diff
@@ -212,6 +212,9 @@ public void handleVerifyYourIdentity() {
     public void handleHowWouldYouLikeToSignIn() {
         // Looks like we sometimes see this UI prompt asking "How would like to sign in?"
         // We press button1, which is "Ok" to confirm the default selection of using device certificate.
-        UiAutomatorUtils.handleButtonClick("android:id/button1", mFindLoginUiElementTimeout);
+        UiAutomatorUtils.handleButtonClickSafely("android:id/button1", mFindLoginUiElementTimeout);
+
+        // Sometimes need to also confirm the selection
```

Comment:
Is this going to attempt two times in every test case? So if there is not a need to confirm then code will still try to click this button?

Reply:
Yes, this probably should be another parameter. I'll add that.